satwatch/docker-compose.yml

```
@@ -0,0 +1,38 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: satwatch_web_1
+      APP_PORT: 8080
+  web:
+    image: ghcr.io/jpcummins/sat.watch:1.1.5@sha256:8fbdd04b01d1ab8fa5626a908622a1dc36fea1c93c24eb24b8213e0bb2a94b5a
+    restart: on-failure
+    stop_grace_period: 1m
+    environment:
+      DATABASE_URL: "postgresql://satwatch:satwatch@db:5432/satwatch?sslmode=disable"
```

Use of the full database service name.
```suggestion
      DATABASE_URL: "postgresql://satwatch:satwatch@satwatch_db_1:5432/satwatch?sslmode=disable"
```